webui/app/static/js/index.js

```diff
@@ -22,7 +22,7 @@ $(document).ready(function () {
             event.preventDefault();
             // show notification and exit handler
             return $(".file-input").notify("Please, select file for uploading", {
-                position: "right",
+                position: ($(window).width() < 992) ? "bottom right" : "right",
```

Comment:
Works good! Will follow up with you separately on doing the same with other notifications (i.e. conversion errors).
![please_select_iphone-6-side-orientation](https://user-images.githubusercontent.com/13045020/33894339-bc37f338-df2b-11e7-9974-415593036913.png)
![errors_iphone-6-side-orientation](https://user-images.githubusercontent.com/13045020/33894340-bc496dd4-df2b-11e7-9874-e6bd2cd8c41a.png)